src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverServiceDaoImpl;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverServiceDaoImpl driverServiceDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverServiceDao.create(driver);
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        return driverServiceDao.get(id);
```

Comment:
```suggestion
    public Drive> get(Long id) {
        return driverServiceDao.get(id);
```
Read common mistakes. you must return not Otional in ServiceImpl